packages/store-ui/src/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,32 @@
+import type { InputHTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  isError?: boolean
+}
+
+const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
+  { isError, ...props },
+  ref
+) {
+  const commonProps = {
```

Comment:
I have mixed feelings about this destructuring part. In line 12 we create an object and spread the `props`. Line 18 we create another object, line 24, we create a third one. All, for spreading in line 29, that, under the hood, will be copied to another one. Isn't it better to add the props directly to the input, like so:
```suggestion
  const state = {
      data-error: isError ? "" : undefined
  }
  return <input data-store-input="" ref={ref}  {...props} {...state} />
```